lib/Interpreter/CppInterOp.cpp

```diff
@@ -1963,8 +1963,12 @@ namespace Cpp {
       // Same applies with member methods which seem to cause parse failures
       // even when we supply the object parameter. Therefore we only use it in
       // cases where we know it works and set this variable to true when we do.
+
+      bool op_flag = !FD->isOverloadedOperator() ||
```

Comment:
Can you add a comment what is the intent with the `op_flag`. To filter out overloaded operators that are not call operators?